_includes/ea/schedule.html

```diff
@@ -0,0 +1,11 @@
+<h3>June 14th</h3>
+<table class="table table-striped">
+  <tr> <td>09:00</td>  <td>Welcome and icebreaker</td> </tr>
+  <tr> <td>09:15</td>  <td>Introduction</td></tr>
```

Comment:
introduction of 50 minutes might be a bit long for the points listed in the lesson:

- What is Conda?
- Why should I use a package and environment management system as part of my research workflow?
- Why use Conda ?

Reply:
What do you think of the following schedule: @SarahAlidoost 
- I separated lesson 3 and 4
- I added a coffee break between them
- I reduced the time for the introduction and moved the coffee break to after lesson 2 (working with environments)
```html
  <tr> <td>09:00</td>  <td>Welcome and icebreaker</td> </tr>
  <tr> <td>09:15</td>  <td>Introduction</td></tr>
  <tr> <td>09:45</td>  <td>Working with environments</td> </tr>
  <tr> <td>10:45</td>  <td>Coffee Break</td></tr>
  <tr> <td>11:00</td>  <td>Using packages and channels</td> </tr>
  <tr> <td>11:30</td>  <td>Coffee Break</td></tr>
  <tr> <td>11:45</td>  <td>Sharing environments</td> </tr>
  <tr> <td>12:45</td>  <td>Wrap-up</td> </tr>
  <tr> <td>13:00</td>  <td>END</td></tr>
```

Reply:
I applied your suggestion.